dotnet/src/Skills/Skills.Grpc/Extensions/GrpcOperationExtensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using Microsoft.SemanticKernel.SkillDefinition;
+using Microsoft.SemanticKernel.Skills.Grpc.Model;
+
+// ReSharper disable once CheckNamespace
+namespace Microsoft.SemanticKernel.Skills.Grpc.Extensions;
+
+/// <summary>
+/// Class for extensions methods for the <see cref="GrpcOperationExtensions"/> class.
+/// </summary>
+internal static class GrpcOperationExtensions
+{
+    /// <summary>
+    /// Returns list of gRPC operation parameters.
+    /// </summary>
+    /// <returns>The list of parameters.</returns>
+    public static IReadOnlyList<ParameterView> GetParameters(this GrpcOperation operation)
```

Comment:
Any special reason for creating an extension method and not adding this to the model itself?

Reply:
The method itself is called getParameters, but it is adding... ?

Reply:
> Any special reason for creating an extension method and not adding this to the model itself?

The idea here is to keep gRPC and SK SDK domains separately and not mix them. It usually simpler to maintain because code is less rigid and changes in one domain does not impacting the other one(s).

Reply:
Yep, it's intentional. To execute a gRPC operation, the address, and payload parameters are required. However, GrpcOperation model class doesn't have them because they are dynamic ones and not described in the .proto files. So, as result we need to create them to be able to resolve them later, from SKContext, before the operation execution.